bootstrap.yml

```diff
@@ -514,6 +515,29 @@ packages:
         environ:
           DESTDIR: '@THIS_COLLECT_DIR@'
 
+  - name: radare2
+    source:
+      subdir: '3rdparty'
+      url: 'https://codeload.github.com/radareorg/radare2/tar.gz/refs/tags/5.4.0'
+      format: 'tar.gz'
+      extract_path: 'radare2-5.4.0'
+      version: '5.4.0'
+    revision: 0
```

Comment:
Revisions start at 1, which is the default if this line is omitted. Usually this line is only added starting from `revision: 2`.